Clarify how redactions affect room state (#2032)
matrix-org/matrix-doc: specification/client_server_api.rst

client receives a redaction event it should change the redacted event in the
same way a server does.

.. NOTE::

    Redacted events can still affect the state of the room. When redacted,

>     state events behave as though their properties were simply not specified

Reviewer: ```suggestion
    state events behave as though their properties were simply not specified,
```

I think this sentence would read better with a comma there

>     unless their properties are protected by the redaction algorithm. For example,

Reviewer 2: ```suggestion
    except those protected by the redaction algorithm. For example,
```
